provider/azure/azure_discover.go

```diff
@@ -2,11 +2,12 @@
 package azure
 
 import (
+	"context"
 	"fmt"
 	"io/ioutil"
 	"log"
 
-	"github.com/Azure/azure-sdk-for-go/arm/network"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2015-06-15/network"
```

Comment:
wow why does this imported package have a date?

Reply:
Unfortunately a bit out of our control. https://github.com/Azure/azure-sdk-for-go/tree/master/services/network/mgmt